spec/support/shared_examples/container.rb

```diff
@@ -487,5 +487,12 @@
         expect(container.resolve(:item)).to eql('item')
       end
     end
+
+    describe 'mixing Strings and Symbols' do
```

Comment:
I'm not a native english speaker but maybe the `describe` needs something that prove the intention of the test, 
something like `allows mixing string and symbols for register and resolving`.

Is just a suggestion 😄

Reply:
I guess you're right, but my intention was to be consistent with existing specs:
https://github.com/dry-rb/dry-container/blob/master/spec/support/shared_examples/container.rb#L246 :)

Reply:
You are right! Maybe in the future, we could revisit the wording in the descriptions. 👍